src/NServiceBus.AcceptanceTests/Performance/When_message_is_audited.cs

```diff
@@ -30,6 +30,26 @@ public class When_message_is_audited : NServiceBusAcceptanceTest
             Assert.IsTrue(context.IsMessageHandledByTheAuditEndpoint);
         }
 
+        [Test]
+        public async Task Should_contain_proper_time_sent()
```

Comment:
In general we prefer one `[Test]` per `when`. If the assertions logically belong to the same `when` then include them in the above test. If not create a new `when`